mala/datahandling/data_converter.py

```diff
@@ -389,6 +407,10 @@ def __convert_single_snapshot(self, snapshot_number,
                     else:
                         output_mesh = output_iteration.meshes[self.target_calculator.target_name]
                         output_mesh.unit_dimension = self.target_calculator.si_dimension
+                        output_mesh.axis_labels = ["x", "y", "z"]
+                        output_mesh.grid_global_offset = [0, 0, 0]
+                        output_mesh.grid_spacing = [1, 1, 1]
+                        output_mesh.grid_unit_SI = 1
```

Comment:
discussed offline: add example for setting a custom geometry with non-orthogonal angles